static/src/javascripts/projects/commercial/modules/measure-timing.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+export const measureTiming = (name: string) => {
+    type TimeTakenInMilliseconds = number;
+    const perf = window.performance;
```

Comment:
I noticed that we use `window.performance` in [another place](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/analytics/google.js#L116) on frontend where we're careful to check it's available before using. [Browser support](https://caniuse.com/?search=performance) for these APIs looks good, so that may be overly cautious now. Might be worth getting the opinion of someone on the Dotcom team about whether we should be doing the same. We also _don't_ check on DCR so I'd be pretty confident this is fine.

Reply:
It looks like it's available on all of our [supported browsers](https://www.theguardian.com/help/recommended-browsers) - having said that Safari is on the cusp (it's available on 8+).

Perhaps it's worth including just to improve backwards compatibility with older browsers, given that the performance impact of the check will be trivial.